ngraph/test/visitors/op/convert.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "util/visitor.hpp"
+
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, convert_op)
+{
+    using Convert = op::v0::Convert;
+
+    NodeBuilder::get_ops().register_factory<Convert>();
+    auto data = std::make_shared<op::Parameter>(element::f32, Shape{2, 4});
+    const element::Type destination_type = element::Type_t::i32;
+
+    const auto convert = std::make_shared<Convert>(data, destination_type);
+    NodeBuilder builder(convert);
+    auto g_convert = as_type_ptr<Convert>(builder.create());
+
+    EXPECT_EQ(g_convert->get_destination_type(), convert->get_destination_type());
+}
```

Comment:
IMHO we should add verification if number of attributes doesn't change. Something like this:
https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/visitors/op/sqrt.cpp#L26-L28

Reply:
Added attribute count check [here](https://github.com/openvinotoolkit/openvino/pull/5533/commits/8595e6bbb100a433e4ccf327928a180f115ca072). I've considered creating new test case for it but decided test it's an overkill.